SingularityUI/app/views/dashboard.coffee

```diff
@@ -41,6 +41,7 @@ class DashboardView extends View
                 requests: pausedRequests
                 haveRequests: pausedRequests.length > 0
                 requestsSubFilter: ''
+                hideNewRequestButton: config.hideNewRequestButton
```

Comment:
This will cause a merge conflict with [Dashboard Page Improvements](https://github.com/HubSpot/Singularity/pull/1023/files#diff-8ede3f844c775e0db1d8b8c0ce9e6150R43). To resolve, simply keep both of the added lines.